### ISSUE TITLE
'make' fails with "No rule to make target 'src/core.e/control.pm6'"

### ISSUE THREAD

**Issue by stuart-little** [rakudo/rakudo#4466]
## The Problem

I am on `Linux x86_64`, currently on the master branch at commit [47ebc86](https://github.com/rakudo/rakudo/commit/47ebc86c98351f5fdb7349642281158bfb79711a). I am trying to build `Rakudo` with

```
perl Configure.pl --gen-moar --gen-nqp --backends=moar
```
(which works fine) and then
```
make
```
This last step fails with the following output:
```
+++ Checking for moar NQP version
+++ Expanding   gen/moar/main-version.nqp
+++ Generating  gen/moar/rakudo.nqp
+++ Generating  gen/moar/Grammar.nqp
+++ Generating  gen/moar/World.nqp
+++ Generating  gen/moar/ModuleLoader.nqp
+++ Compiling   blib/Perl6/ModuleLoader.moarvm
+++ Generating  gen/moar/Ops.nqp
+++ Compiling   dynext/libperl6_ops_moar.so
+++ Compiling   blib/Perl6/Ops.moarvm
+++ Generating  gen/moar/Pod.nqp
+++ Compiling   blib/Perl6/Pod.moarvm
+++ Compiling   blib/Perl6/World.moarvm
+++ Generating  gen/moar/Actions.nqp
+++ Compiling   blib/Perl6/Actions.moarvm
+++ Compiling   blib/Perl6/Grammar.moarvm
+++ Generating  gen/moar/Compiler.nqp
+++ Generating  gen/moar/Optimizer.nqp
+++ Compiling   blib/Perl6/Optimizer.moarvm
+++ Compiling   blib/Perl6/Compiler.moarvm
+++ Generating  gen/moar/SysConfig.nqp
+++ Compiling   blib/Perl6/SysConfig.moarvm
+++ Compiling   rakudo.moarvm
+++ Generating  gen/moar/BOOTSTRAP/v6c.nqp
+++ Generating  gen/moar/Metamodel.nqp
+++ Compiling   blib/Perl6/Metamodel.moarvm
+++ Compiling   blib/Perl6/BOOTSTRAP/v6c.moarvm
+++ Compiling   blib/CORE.c.setting.moarvm
The following step can take a long time, please be patient.
Stage start      :   0.000
Stage parse      :  54.941
Stage syntaxcheck:   0.000
Stage ast        :   0.000
Stage optimize   :   5.797
Stage mast       :  11.209
Stage mbc        :   2.203
+++ Generating  gen/moar/BOOTSTRAP/v6d.nqp
+++ Compiling   blib/Perl6/BOOTSTRAP/v6d.moarvm
+++ Compiling   blib/CORE.d.setting.moarvm
The following step can take a long time, please be patient.
Stage start      :   0.000
Stage parse      :   0.229
Stage syntaxcheck:   0.000
Stage ast        :   0.000
Stage optimize   :   0.016
Stage mast       :   0.041
Stage mbc        :   0.005
+++ Generating  gen/moar/BOOTSTRAP/v6e.nqp
+++ Compiling   blib/Perl6/BOOTSTRAP/v6e.moarvm
make: *** No rule to make target 'src/core.e/control.pm6', needed by 'blib/CORE.e.setting.moarvm'.  Stop.
```
I have tried on a couple of different machines, both `Linux x86_64`. 

## Expected Behavior

A normal build.

## Actual Behavior

The failure reported above. 

## Steps to Reproduce

See the build steps described above. 

## Environment
* Operating system: `Linux x86_64` (`Ubuntu 20.04`)
* Compiler version (`perl6 -v` or `raku -v`):
```
$ raku -v
Welcome to 𝐑𝐚𝐤𝐮𝐝𝐨™ v2021.06-58-g38626c686.
Implementing the 𝐑𝐚𝐤𝐮™ programming language v6.d.
Built on MoarVM version 2021.06-16-g74e828f0a.
```

**Comment by vrurg** [comment]
Thank you! I think @lizmat just forgot to add the file to the repository.

**Comment by stuart-little** [comment]
I think so: `src/core.e/control.pm6` doesn't seem to be there :)..

**Comment by lizmat** [comment]
Fixed with a2faac4bd817c61f6dbae
